com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Camera/HDAdditionalCameraData.cs

```diff
@@ -360,6 +360,12 @@ public enum TAAQualityLevel
         /// <summary>Enable to retain history buffers even if the camera is disabled.</summary>
         public bool hasPersistentHistory = false;
 
+        /// <summary>Screen size used when Screen Coordinates Override is active.</summary>
+        public Vector4 screenSizeOverride;
```

Comment:
tag for @RSlysz

Reply:
This should have no impact on the refactor we are doing